trappy/plotter/ILinePlotGen.py

```diff
@@ -34,6 +34,18 @@
 
 from IPython.display import display, HTML
 
+def df_to_dygraph(data_frame):
```

Comment:
Not sure what happens to the order of the labels though (we use `OrderedDict()` in `_plot()`). Maybe we'll have to put it inside an `OrderedDict()`...